python/qisrc/test/test_qisrc_push.py

```diff
@@ -38,7 +38,7 @@ def test_not_under_code_review_with_no_review(qisrc_action, git_server):
     (_, remote) = foo_git.call("ls-remote", "origin", "master", raises=False)
     assert remote == "%s\trefs/heads/master" % sha1
 
-def test_using_dash_y(qisrc_action, git_server):
+def test_using_dash_y_and_dash_f(qisrc_action, git_server):
```

Comment:
I don't think so, but it's only 3 lines duplicated, not really a problem in the tests